SeaPublicWebsite.BusinessLogic/SeaPublicWebsite.BusinessLogic.csproj

```diff
@@ -6,7 +6,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="CabinetOffice.GovUkDesignSystem" Version="1.0.0-d16cb08" />
+      <PackageReference Include="CabinetOffice.GovUkDesignSystem" Version="1.0.0-1dd1b5d" />

```

Comment:
This is both in here and in `SeaPublicWebsite.csproj`. The main website project depends on the BusinessLogic, so I can't make this depend on the website (but I can delete this packageReference tag from the main website project)

Reply:
Fine to leave this here until (if) we delete the enum attributes from this project